### ISSUE TITLE
Allow the username to be updated

### ISSUE THREAD

**Issue by bhdouglass** [ubports/ubuntu-touch#855]
Currently the username does not appear to be editable anywhere in the setting (called the "Preferred Name" in the welcome wizard). It would be great to provide the user an option to change the name.

**Comment by Flohack74** [comment]
See also #231 - it would be important to make a generic solution, that is compatible with upstream as far as possible.

**Comment by Flohack74** [comment]
Just thinking now, is this a duplicate of my ticket? =)

**Comment by bhdouglass** [comment]
I saw that, but are we talking about the same thing?

**Comment by Flohack74** [comment]
Where is preferred name visible?

**Comment by Flohack74** [comment]
Yes its the same, we need to update `/etc/machine-info` upon user request.

**Comment by bhdouglass** [comment]
@Flohack74 this is the contents of `/etc/machine-info` on my phone:
```
PRETTY_HOSTNAME="MX4 Ubuntu Edition"
CHASSIS=handset
```

Where as my preferred name that is shown on the unlock screen is "Brian", so it doesn't appear that we are talking about the same thing.

**Comment by Flohack74** [comment]
Interesting... At least machine info is used for the Bluetooth name and should be synced with the device name you enter. Maybe you can investigate a little bit more, we got 0 documents about how this works ;)
